### PR TITLE
feat: add html cached version

### PR DESCRIPTION
**Pull request by GabrielBdeC**
Fixes # .

## Changes proposed in this pull request

-
-

To test (it takes a while): `npm install github:<username>/venom#<branch>`

### CODE REVIEW

File: src/api/whatsapp.ts [hunk 1]
```diff
@@ -43,7 +43,8 @@ export class Whatsapp extends ControlsLayer {
       let useWebpack = false
       if (
         this.options.forceWebpack === false &&
-        this.options.webVersion === false
+        this.options.webVersion &&
```
Comment (CBeraldo) [inline src/api/whatsapp.ts:46]: Confirmar force webpack false/true

File: src/controllers/browser.ts [hunk 2]
```diff
@@ -56,18 +57,15 @@ export async function initWhatsapp(
 
     const { userPass, userProxy, addProxy } = options
 
-    // TODO Create own cache version and download it or use HTML from options
-    if (typeof options.webVersion === 'string' && options.webVersion.length) {
+    if (
```
Comment (CBeraldo) [inline src/controllers/browser.ts:60]: whatsappCacheManager/service
 ! return null
this.existsWAVErsionHTML
this.existsWAVErsion

**Review by CBeraldo — CHANGES_REQUESTED** [review]

File: src/controllers/browser.ts [hunk 3]
```diff
@@ -49,44 +47,17 @@ export async function initWhatsapp(
   if (!waPage) {
     return false
   }
+
+  whatsappCacheManagement.setup(waPage, options)
```
Comment (vinnydeveloper) [inline src/controllers/browser.ts:51]: Colocar dentro do trycatch

```suggestion
  whatsappCacheManagement.setup(waPage, options)
```

File: src/controllers/whatsapp-cache-management.ts [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+class WhatsappCacheManagement {
+  private baseWppCacheURL =
+    'https://raw.githubusercontent.com/wppconnect-team/wa-version/main/html'
+  private waPage: Page | undefined
+  private waVersionHTML: string = ''
+  private webVersion: string = ''
+  private forceWebpack: boolean = false
+
+  async setup(waPage: Page, options: options | CreateConfig) {
```
Comment (vinnydeveloper) [inline src/controllers/whatsapp-cache-management.ts:16]: Não é necessário o async

```suggestion
  async setup(waPage: Page, options: options | CreateConfig) {
```

**Review by vinnydeveloper — CHANGES_REQUESTED** [review]

**Review by vinnydeveloper — APPROVED** [review]

**Pull request merged**